### ISSUE TITLE
[Artist] get/set user specific data through metaphysics.

### ISSUE THREAD

**Issue by alloy** [artsy/emission#15]
- [ ] Get artist follow status
- [ ] Update artist follow status

**Comment by sarahscott** [comment]
This is done, right?

**Comment by alloy** [comment]
Nope, this still needs to be done and is a great candidate for people that want to play with GraphQL and specifically mutations 👍
